<?xml version='1.0' encoding='UTF-8'?>
<root version="2.3.8">
 <tabbed_widget parent="main_window" name="Main Window">
  <Tab containers="1" tab_name="tab1">
   <Container>
    <DockSplitter orientation="-" sizes="0;0.5;0.5" count="3">
     <DockArea name="...">
      <plot mode="TimeSeries" style="Lines">
       <range top="-0.582185" left="30.793113" right="46.905788" bottom="-19.432888"/>
       <limitY/>
       <curve name="/ati_ft_data/wrench/force/z" color="#1f77b4"/>
      </plot>
     </DockArea>
     <DockSplitter orientation="|" sizes="0.500715;0.499285" count="2">
      <DockArea name="...">
       <plot mode="TimeSeries" style="Lines">
        <range top="0.465471" left="30.793113" right="46.905788" bottom="-23.788352"/>
        <limitY/>
        <curve name="/ati_ft_data/wrench/force/z" color="#1f77b4"/>
       </plot>
      </DockArea>
      <DockArea name="...">
       <plot mode="TimeSeries" style="Lines">
        <range top="27.832634" left="30.793113" right="46.905788" bottom="10.605775"/>
        <limitY/>
        <curve name="/controller/data_analysis/force_error" color="#d62728"/>
       </plot>
      </DockArea>
     </DockSplitter>
     <DockSplitter orientation="|" sizes="0.500715;0.499285" count="2">
      <DockArea name="...">
       <plot mode="TimeSeries" style="Lines">
        <range top="19.394225" left="31.805438" right="46.932858" bottom="2.167366"/>
        <limitY/>
        <curve name="/controller/data_analysis/current_force" color="#1ac938"/>
       </plot>
      </DockArea>
      <DockArea name="...">
       <plot mode="TimeSeries" style="Lines">
        <range top="0.003562" left="30.793113" right="46.905788" bottom="0.003499"/>
        <limitY/>
        <curve name="/controller/data_analysis/ctrl_output" color="#ff7f0e"/>
       </plot>
      </DockArea>
     </DockSplitter>
    </DockSplitter>
   </Container>
  </Tab>
  <currentTabIndex index="0"/>
 </tabbed_widget>
 <use_relative_time_offset enabled="1"/>
 <!-- - - - - - - - - - - - - - - -->
 <!-- - - - - - - - - - - - - - - -->
 <Plugins>
  <plugin ID="DataLoad CSV">
   <default time_axis=""/>
  </plugin>
  <plugin ID="DataLoad ROS bags">
   <use_header_stamp value="false"/>
   <use_renaming_rules value="true"/>
   <discard_large_arrays value="true"/>
   <max_array_size value="100"/>
  </plugin>
  <plugin ID="DataLoad ULog"/>
  <plugin ID="ROS Topic Subscriber">
   <use_header_stamp value="false"/>
   <use_renaming_rules value="false"/>
   <discard_large_arrays value="true"/>
   <max_array_size value="100"/>
  </plugin>
  <plugin ID="UDP Server"/>
  <plugin ID="WebSocket Server"/>
  <plugin ID="ZMQ Subscriber"/>
  <plugin status="idle" ID="CSV Exporter"/>
  <plugin status="idle" ID="ROS /rosout Visualization"/>
  <plugin status="idle" ID="ROS Topic Re-Publisher"/>
 </Plugins>
 <!-- - - - - - - - - - - - - - - -->
 <previouslyLoaded_Datafiles/>
 <!-- - - - - - - - - - - - - - - -->
 <customMathEquations/>
 <snippets/>
 <!-- - - - - - - - - - - - - - - -->
</root>

